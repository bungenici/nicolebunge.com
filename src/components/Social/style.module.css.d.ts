/* eslint-disable import/prefer-default-export */

export const social: string;
