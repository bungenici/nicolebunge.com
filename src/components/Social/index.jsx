import React from 'react';
import Icons from '../Icons';
import styles from './style.module.css';

const Social = () => (
  <div className={styles.social}>
    <Icons />
  </div>
);

export default Social;
