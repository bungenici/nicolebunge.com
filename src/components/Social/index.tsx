import classNames from 'classnames';
import React from 'react';
import Icons from '../Icons';
import styles from './style.module.css';

export interface SocialProps {
  className?: string;
}

const Social: React.FC<SocialProps> = (props) => {
  const { className, ...otherProps } = props;

  return (
    <div className={classNames(className, styles.social)} {...otherProps}>
      <Icons />
    </div>
  );
};

export default Social;
