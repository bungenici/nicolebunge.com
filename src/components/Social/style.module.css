.social {
  background-color: var(--color-secondary);
  background-image: linear-gradient(to right, #f8f8f8 1px, transparent 1px);
  background-position: 50%;
  background-size: calc(var(--grid-max-width) / 3);
  bottom: 0;
  display: flex;
  flex-wrap: wrap;
  justify-content: flex-end;
  left: 0;
  padding: 1.5em 3.5em;
  position: fixed;
  right: 0;
  will-change: transform;
  z-index: 1;
}
