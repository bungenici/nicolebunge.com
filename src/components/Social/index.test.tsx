/* eslint-disable import/no-extraneous-dependencies */

import React from 'react';
import renderer from 'react-test-renderer';
import Social from '.';

describe('Social component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Social />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
