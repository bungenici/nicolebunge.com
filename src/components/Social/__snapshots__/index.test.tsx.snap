// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Social component renders correctly 1`] = `
<div
  className="social"
>
  <ul
    className="icons"
  >
    <li>
      <a
        href="https://www.instagram.com/nicole.bunge/"
        itemProp="sameAs"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="Nicole Bunge auf Instagram"
          src="test-file-stub"
        />
      </a>
    </li>
    <li>
      <a
        href="https://www.linkedin.com/in/nicole-bunge-1b891415a/"
        itemProp="sameAs"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="Nicole Bunge auf LinkedIn"
          src="test-file-stub"
        />
      </a>
    </li>
    <li>
      <a
        href="https://www.xing.com/profile/Nicole_Bunge3"
        itemProp="sameAs"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="Nicole Bunge auf Xing"
          src="test-file-stub"
        />
      </a>
    </li>
  </ul>
</div>
`;
